source/features/improve-shortcut-help.tsx

```diff
@@ -30,18 +26,9 @@ function improveShortcutHelp(dialog: Element): void {
 	);
 }
 
-function fixKeys(dialog: Element): void {
-	for (const key of select.all('kbd', dialog)) {
-		if (key.textContent!.includes(' ')) {
-			key.replaceWith(...splitKeys(key.textContent!));
-		}
-	}
-}
-
 const observer = new MutationObserver(([{target}]) => {
 	if (target instanceof Element && !select.exists('.js-details-dialog-spinner', target)) {
 		improveShortcutHelp(target);
-		fixKeys(target);
```

Comment:
Fix #4962. The reason I ["Can't replicate now"](https://github.com/refined-github/refined-github/issues/4962#issuecomment-953206637) is that this feature stopped working completely as the command palette was added.